File: t.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    class Program
    {
        static int fibonacci(int n)
        {
            if (n < 2)
                return n;
            return fibonacci(n - 2) + fibonacci(n - 1);
        }
        static void Main(string[] args)
        {
            var start = DateTime.Now;
            Console.WriteLine("fibn: {0}, time: {1}", fibonacci(42), (DateTime.Now - start).TotalMilliseconds);
            Console.ReadKey();
        }
    }
}

File: t.rs

use std::time::{SystemTime};

fn fibonacci(i: i32) -> i32
{
	if i < 2 {
		return i;
	}
	return fibonacci(i - 2) + fibonacci(i - 1);
}
 
fn main()
{
	let start = SystemTime::now();
	let fibn = fibonacci(45);
	println!("{:?}", fibn);
	match start.elapsed() {
		Ok(elapsed) => {
			println!("elapsed: {}s{}ms", elapsed.as_secs(), elapsed.subsec_nanos()/1000000);
		}
		Err(e) => {
			println!("{:?}", e);
		}
	}
}
